spec/inherited_class_var_spec.rb

```diff
@@ -34,32 +34,6 @@ class InheritedBaseClass; end
         expect(Parent.inherited_hash).to eql(test1: 'test1', test2: 'test2')
         expect(Parent.inherited_hash.object_id).to eql Parent.inherited_hash.object_id
       end
-
-      context 'with cache cleared' do
-        before do
-          # NOTE deep_clear_class_cache doesn't work here...
```

Comment:
i think you were confused about what the caching does. clearing the cache won't change the values of the `inherited_class_var`, it just forces a recalculation in the block: https://github.com/FinalCAD/inherited_class_var/blob/e6a20ce15933dbec12e1a3be8a2c2b6fbcf95b41/lib/inherited_class_var.rb#L72-L76

i wrote a comment to explain why i wrote the caching code here: https://github.com/FinalCAD/inherited_class_var/pull/3/commits/9909414127e1deee9505f4399a6dc9707b0056f4

so i don't think this code is needed.